Architecture.md

```diff
@@ -0,0 +1,58 @@
+# Architecture
```

Comment:
These files are useful. You might like to take a look at the SourceForge wiki to see what to incorporate (https://sourceforge.net/p/minsky/wiki/Home/)